st2common/st2common/BUILD

```diff
@@ -1 +1,11 @@
-python_sources()
+python_sources(
+    dependencies=[
+        ":openapi_spec",
+    ]
+)
+
+# These may be loaded with st2common.util.spec_loader
+resources(
+    name="openapi_spec",
+    sources=["openapi.yaml", "openapi.yaml.j2"],
+)
```

Comment:
A future PR will add `pants-plugins/api_spec` which will wire this up so that it gets regenerated whenever someone runs the `fmt` goal with something like: `./pants fmt st2common/st2comon/openapi.yaml`. That will take advantage of the fine-grained caching so that `openapi.yaml` always gets updated when the files used to generate it change.